cmd-info.c

```diff
@@ -785,6 +785,11 @@ int command_info(int argc, char *argv[], struct opts *opts)
 			.flags = SYMTAB_FL_DEMANGLE,
 		};
 
+		if (!opts->exename) {
+			pr_out("Usage: uftrace info [--symbols] [COMMAND]\n");
```

Comment:
Note that the `[xxx]` sign means that using xxx is optional.  In this case using `--symbols` is not optional.  And I prefer `pr_use` for usage mistakes.

Also I think this message is meaningless since it's only showed when there's no data file which is already warned.  Or maybe you can delay the "no data" warning after checking symbols option.